precise/poisson_generator_ps.h

```diff
@@ -123,8 +123,9 @@ class poisson_generator_ps : public DeviceNode
    */
   struct Parameters_
   {
-    double rate_;      //!< process rate [Hz]
-    double dead_time_; //!< dead time [ms]
+    double rate_;       //!< process rate [Hz]
+    double dead_time_;  //!< dead time [ms]
+    bool rate_changed_; //!< flag for rate change
```

Comment:
I am not entirely happy with `rate_changed_` here in `P_`, since it is not a model parameter but an implementation detail. `B_` is probably the best place to put it, since `V_` is for variables recomputed by `calibrate()`.  Would it be difficult to move the flag to `B_`?